auth-web/src/components/auth/TestimonialQuotes.vue

```diff
@@ -1,19 +1,47 @@
 <template>
   <section id="testimonials-container">
     <v-container class="quote-container">
-      <p>"We're incredibly excited to be able to incorporate as a Benefit Company in BC, it really reflects our values
-        and the role we want to play."</p>
-      <p><strong>&ndash; BC Benefit Company user</strong></p>
+      <div :class="{ 'fadeOut': isFading, 'fadeIn': !isFading}">
```

Comment:
Isn't this what transitions are for? :)

Reply:
ahh, i'll look into that :)

Reply:
AS stated below, i just used base css the transitions, simplified it a bit though. Let me know what you think. I didn't find vuetify transitions all that intuitive.

Reply:
Haha, now you know about key frame transitions.

I was actually thinking about this: https://vuetifyjs.com/en/styles/transitions/

Reply:
... which we already use in other places.

A good example would be the COD view where I use v-transitions to soften the change between the edit page and the review page.

Reply:
Anyway, what you've done now is way better and sufficient IMHO.

Reply:
After you mentioned doing it with browser transitions, i had to try!
Thanks for that.

Reply:
Resolved!

Reply:
> After you mentioned doing it with browser transitions, i had to try!
> Thanks for that.

YW. Sounds like you had fun :)